src/blocks/author-carousel/edit.js

```
@@ -1,286 +1,397 @@
+import { AuthorBlockPlaceholder, AuthorPicker } from '../../js/components';
 import AuthorCarouselPreview from './components/AuthorCarouselPreview';
 import CarouselLayoutSelector from './components/CarouselLayoutSelector';
-import {shuffle} from '@wordpress/icons';
+import { shuffle } from '@wordpress/icons';
 
 /**
  * The edit function for the Author Carousel block.
  *
- * @param {Object} props Block properties.
+ * @param {Object} props               Block properties.
+ * @param          props.attributes
```

ESLint

[error] Missing JSDoc @param "props.attributes" type.

(jsdoc/require-param-type)